### PR TITLE
Add a way to disable mouse move, pan or zoom

### LINKED ISSUE #163: Enable/Disable Zoom
Hi,

I saw that you removed `EventCapture` https://github.com/rrag/react-stockcharts/blob/78bbf30e4e29580aceedbc0e6fe0c8829b95b88b/docs/md/CHANGE-LOG.md

Is there any way now to enable/disable the zoom on demand? The problem being when you scroll down a page and happen to be on top of a chart then it starts zooming instead of scrolling. We want to have a button to enable chart interaction instead of having it by default.

Thanks, we <3 your lib :)

### PR DESCRIPTION
**Pull request by shprink**
Closes #163

### CODE REVIEW

File: docs/md/ZOOM-AND-PAN.md [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+    ...
+    disableMouseMoveEvent={false}
+    disablePanEvent={false}
+    disableZoomEvent={false}
+    ...
+/>
 ```
-enabling `zoom` and `pan` to `true`
 
 By default the chart is not focused. meaning. you have to click on the chart to get focus, and only then will the scroll events trigger a zoom.
```
Comment (rrag) [inline docs/md/ZOOM-AND-PAN.md]: Thanks for fixing the docs too, can you remove

```diff
- By default the chart is not focused. meaning. you have to click on the chart to get focus, and only then will the scroll events trigger a zoom.
```

**Comment by shprink** [comment]
done

**Comment by rrag** [comment]
Thank you. will rebuild and publish the docs site later

**Comment by rrag** [comment]
Thanks for the PR

published new beta version, install again as `npm i -S react-stockcharts@next`

I had forgotten about the zoom capabilities by click & drag of the x and y axis, fixed that also, look at `zoomEnabled` prop in `XAxis`, `YAxis` in [CandleStickChartWithZoomPan.jsx](https://github.com/rrag/react-stockcharts/blob/master/docs/lib/charts/CandleStickChartWithZoomPan.jsx)

**Pull request merged**